test/built-ins/Object/getOwnPropertyDescriptors/inherited-properties-omitted.js

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2016 Jordan Harband. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Object.getOwnPropertyDescriptors does not see inherited properties.
+es7id: pending
+author: Jordan Harband
+---*/
+
+var F = function G() {};
+F.prototype.a = {};
+F.prototype.b = {};
+
+var f = new F();
+f.b = {}; // shadow the prototype
+Object.defineProperty(f, 'c', {
+  enumerable: false,
+  configurable: true,
+  writable: false,
+  value: {}
+}); // solely an own property
+
+var result = Object.getOwnPropertyDescriptors(f);
+
+assert.sameValue(!!result.b, true, 'b has a descriptor');
+assert.sameValue(!!result.c, true, 'c has a descriptor');
+
+assert.sameValue(result.b.enumerable, true, 'b is enumerable');
+assert.sameValue(result.b.configurable, true, 'b is configurable');
+assert.sameValue(result.b.writable, true, 'b is writable');
+assert.sameValue(result.b.value, f.b, 'b’s value is f.b');
```

Comment:
As written, it's possible to pass this assertion by returning the "shadowed"
value of `b`. It seems highly unlikely that an implementor would get
"enumerable", "configurable", and "writable" correct but make an error on
"value", to be sure. All the same (to be extra cautious), I'd recommend
creating a binding for the "shadowing" value and expressing this expectation
with that reference.